src/config.rs

```diff
@@ -1104,6 +1104,78 @@ pub mod log_level_serde {
     }
 }
 
+#[derive(Clone, Copy, Serialize, Deserialize, PartialEq, Debug)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct YatpConfig {
+    pub min_thread_count: usize,
+    pub max_thread_count: usize,
+    // FIXME: Add more configs when they are effective in yatp
+}
+
+impl YatpConfig {
+    fn validate(&self) -> Result<(), Box<dyn Error>> {
+        // FIXME: Replace "readpool" prefix in error messages when it is used
+        // elsewhere.
+
+        if self.min_thread_count == 0 {
+            return Err("readpool.unified.min-thread-count should be > 0"
+                .to_string()
+                .into());
+        }
+        if self.max_thread_count < self.min_thread_count {
+            return Err(
+                "readpool.unified.max-thread-count should be >= readpool.unified.min-thread-count"
+                    .to_string()
+                    .into(),
+            );
+        }
+        Ok(())
+    }
+}
+
+const YATP_READPOOL_MIN_CONCURRENCY: usize = 4;
+
+// FIXME: Support other default configs if yatp is used elsewhere besides
```

Comment:
I think serde will use the default defined here instead if it's used elsewhere. That's one of the reasons why there are so many configuration structs for rocksdb CF options.

Reply:
I change the name of the config to `UnifiedReadPoolConfig` and provide a clearer comment.